main.tex

```diff
@@ -164,15 +164,15 @@ \subsection{Credentials}
 
 The coordinator issues anonymous credentials which authenticate attributes in response to registration requests. We use keyed-verification anonymous credentials (introduced in~\cite{chase2014algebraic}), in particular the scheme from~\cite{chase2019signal} which supports group attributes (attributes whose value is an element of the underlying group $\mathbb{G}$). A user can then prove possession of a credential in zero knowledge in a subsequent registration request, without the coordinator being able to link it to the registration from which it originates.
 
-In order to facilitate construction of a CoinJoin transaction while protecting the privacy of participants we instantiate the scheme with a single group attribute $M_a$ which encodes a confidential Bitcoin amount as a Pedersen commitment. These commitments are never opened, instead properties of the underlying values are proven in zero knowledge, allowing the coordinator to verify validity of the requests made by honest participants. Ideally the coordinator would not learn anything that can't be learned from the resulting CoinJoin transaction.
+In order to facilitate construction of a CoinJoin transaction while protecting the privacy of participants, we instantiate the scheme with a single group attribute $M_a$ which encodes a confidential Bitcoin amount as a Pedersen commitment. These commitments are never opened. Instead, properties of the values they commit to are proven in zero knowledge, allowing the coordinator to validate the requests made by honest participants. Ideally the coordinator does not learn anything that can't be learned from the resulting CoinJoin transaction.
```

Comment:
"ideally the coordinator does not learn" seems a bit strong to me in the present tense, we can't actually guarantee that, especially when considering denial of service or other failures when a coinjoin tx isn't even produced, or things like de-registration, which reveals information to a coordinator about an input that doesn't end up participating

Reply:
What about "Ideally the coordinator ... CoinJoin Transaction. But in practice when considering DoS or other failure that might cause a CoinJoin not to be produced some information about an input that does not participate may be shared" ... and explain how that effects privacy? Maybe too long winded

Reply:
see #48 but that's probably not going to make it into this as per discussion...

it's not wrong, ideally that would be the case, but it'd be better if it didn't imply that this was accomplished